src/main/java/com/example/SpringCRUD/service/EmployeeService.java

```diff
@@ -0,0 +1,75 @@
+package com.example.SpringCRUD.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.example.SpringCRUD.Repository.EmployeeRepository;
+import com.example.SpringCRUD.entity.EmployeeEntity;
+import com.example.SpringCRUD.exception.RecordNotFoundException;
+
+@Service
+public class EmployeeService {
+
+@Autowired
+EmployeeRepository employeeRepository;
+
+ public List<EmployeeEntity> getAllEmployees(){
+	List<EmployeeEntity> employeeList = employeeRepository.findAll();
+	if(employeeList.size() > 0)
+	{
+		return employeeList;
+	}
+	else
+	{
+		return new ArrayList<EmployeeEntity>();
+	}
+	
+}
+
+public EmployeeEntity getEmployeeById(Long Id) throws RecordNotFoundException{
+	Optional<EmployeeEntity> List = employeeRepository.findById(Id);
```

Comment:
Please follow the camel case for variable naming convention

Reply:
Done